flashinfer/cute_dsl/gated_delta_rule.py

```
@@ -0,0 +1,1823 @@
+        >>> b = torch.randn(B, T, HV, device='cuda', dtype=torch.bfloat16)
+        >>> A_log = torch.randn(HV, device='cuda', dtype=torch.float32)
+        >>> dt_bias = torch.randn(HV, device='cuda', dtype=torch.float32)
+        >>> h_state = torch.randn(B, HV, V, K, device='cuda', dtype=torch.bfloat16)
+        >>> output = gated_delta_rule(
+        ...     A_log, a, dt_bias, q=q, k=k, v=v, b=b,
+        ...     initial_state_source=h_state
+        ... )
+    """
+    global _compiled_kernels
+
+    B, T, H, K = q.shape
```

Ruff (0.14.14)

[warning] Unpacked variable `H` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)